config/shared/activities.js

```diff
@@ -54,6 +55,10 @@ function getSchemaId(type) {
   return type.includes('/') && first(type.split('/'));
 }
 
+function getTypeName(type) {
+  return type.includes('/') && type.split('/')[1];
+}
```

Comment:
It's very similar to `getSchemaId`.
How about `parseType` that will return `{ schema, typeName }` and will be re-used in both?